README.md

```diff
@@ -21,7 +26,7 @@ jobs:
     - name: Checkout
       uses: actions/checkout@v2
     - name: Set up V version 0.1.24
-      uses: nocturlab/setup-vlang-action@1.0.0
+      uses: nocturlab/setup-vlang-action@1.0.1
```

Comment:
Probably, v1 fits better.

https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsuses